internal/pkg/update/repository.go

```diff
@@ -13,7 +13,7 @@ type Repository interface {
 	GetAvailableReleaseNotesVersions(name string) (version.Collection, error)
 	// Downloads the versioned package to download dir to downloadDir.
 	// Returns the full path to the downloaded package, the download size in bytes, or an error if one occurred.
-	DownloadVersion(name, version, downloadDir string) (downloadPath string, downloadedBytes int64, err error)
```

Comment:
is this assuming that path will always be `/bin`?

Reply:
The new package updates the binary in-place, so it will use whatever directory the binary is currently in.